charts/plex/2.0.0/values.yaml

```
@@ -0,0 +1,73 @@
+# Default values for plex-media-server.
+
+image:
+  repository: k8sathome/plex-media-server
```

```suggestion
  repository: plexinc/pms-docker
```

Alternate `linuxserver/plex`

I don't use Plex, so I'm not sure which docker image is preferred. Other options are also available.